.travis.yml

```diff
@@ -5,9 +5,8 @@ os:
   - osx
   - windows
 julia:
-  - 1.1
```

Comment:
This change doesn't really match the commit message.

Reply:
`s/1.3/1.1/g` :man_shrugging: